docs/docs/resources.md

```diff
@@ -0,0 +1,403 @@
+# Resources
+
+_this document has been adapted/copied[^copying] from the Starbeam[^starbeam] documentation_
+
+
+This is a highlevel introduction to Resources, and how to use them.
+For how to integrate Resources in to Ember (Components, etc), see ['./ember.md]('./ember.md'); 
+
+In addition to the live demos accompanying each code snippet, all code snippets will have their starbeam counterparts below them, so that folks can see how similar the libraries are.
+
+When Starbeam is integrated in to Ember, there will be a codemod to convert from ember-resources' APIs to Starbeam's APIs. 
+
+_details on that soon_
+
+
+<hr>
+
+
+> **Note** <br>
+> A resource is a reactive function with cleanup logic.
+
+Resources are created with an owner, and whenever the owner is cleaned up, the resource is also cleaned up. This is called ownership linking.
+
+Typically, a component in your framework will own your resources. The framework renderer will make sure that when your component is unmounted, its associated resources are cleaned up.
+
+<details>
+<summary>Resources Convert Processes Into Values</summary>
+
+Typically, a resource converts an imperative, stateful process, such as an asynchronous request or a ticking timer, into a reactive value.
+
+That allows you to work with a process just like you'd work with any other reactive value.
+
+This is a very powerful capability, because it means that adding cleanup logic to an existing reactive value doesn't change the code that works with the value.
+
+The only thing that changes when you convert a reactive value into a resource is that it must be instantiated with an owner. The owner defines the resource's lifetime. Once you've instantiated a resource, the value behaves like any other reactive value.
+
+</details>
+
+## A Very Simple Resource
+
+To illustrate the concept, let's create a simple resource that represents the current time.
+
+```js 
+import { cell, resource } from "ember-resources";
+ 
+export const Now = resource(({ on }) => {
+  const now = cell(Date.now());
+ 
+  const timer = setInterval(() => {
+    now.set(Date.now());
+  });
+ 
+  on.cleanup(() => {
+    clearInterval(timer);
+  });
+ 
+  return now;
+});
+```
+
+To see this code in action, [checkout the live demo](https://limber.glimdown.com/edit?c=MQAggiDKAuD2AOB3AhtAxgCwFBbbAJgKYgBmhAdmsdMgOYDOIay5IARsQK72H4hwh8ASwBOhNNAA2ATxBjyREU1gBbeLHIVo9ADRZkjRIUmSQBuRUX8MxPERD1yQ%2BPELRrxSUIButgn5VCRhIRVQA6HAB5eGghDXoALhwAWhAAAy9fNJBUgH15RUYhVnhJZCpcrFSMn2J4MW8hQkRs-MtCESKSsqodEFLy4toPB1hOESozEmgOkcziAo7K6vn%2BhqbEdmNYFpA2hQ6u-p7CPoG0IbnaiwOlDkkdkfoxicJKrDTP2gArRnmsIRqWAidwAbwsz3GvSYxlMAF9SKEVCAAOSEFQcETJMSQ170FEAbhwhAAHuoQcpyPR3AA5R4AXghLyoAAoWeCNCA4QBKED0gB8IFBWBAlOpIHIDJhJhZABFUIQwpLECzudyiSARWL3LFAkpGTxoABJcgzETeZCSNm8gVCrWi5VhQ1yhVKnaq9VankarUaMJoSSEFiceDWvmC4Wi0UBoMiE1mi1W3UdT2i71arViaDjVjKonpnAAHhmajKM35WqN7iEiRAxcBhH5oNBdMQcLhhYA9MmK12SwNyx9PkA&format=glimdown).
```

Comment:
This demo doesn't work:

![image](https://github.com/NullVoxPopuli/ember-resources/assets/9835/5a33ab06-1b2f-4009-be2a-332829d62786)

Also, it's unclear to me what the "options" it is talking about are.

Reply:
demo will work when the PR is merged :sweat_smile:  -- because it implements the needed feature where Cells are directly renderable (via the smallest helper manager ever :tada: )

Reply:
> Also, it's unclear to me what the "options" it is talking about are.

hm, yeah, it's a little orthogonal -- but it's how to use code fences on https://limber.glimdown.com -- maybe that should be removed and extracted to a separate set of docs